pkg/apis/servicecatalog/checksum/unversioned/checksum.go

```diff
@@ -64,3 +64,16 @@ func BindingSpecChecksum(spec servicecatalog.BindingSpec) string {
 	sum := sha256.Sum256([]byte(specString))
 	return fmt.Sprintf("%x", sum)
 }
+
+// BrokerSpecChecksum calculates a sha256 hash for the given BrokerSpec based on
+// the following fields:
+// - URL
+// - AuthInfo (may be nil, but special handling is unnecessary with %v)
+func BrokerSpecChecksum(spec servicecatalog.BrokerSpec) string {
+	specString := ""
+	specString += fmt.Sprintf("URL: %v\n", spec.URL)
+	specString += fmt.Sprintf("AuthInfo: %v\n", spec.AuthInfo)
```

Comment:
This is the line that's produced (test input updated): AuthInfo: &{&ObjectReference{Kind:,Namespace:test-ns,Name:test-secret,UID:,APIVersion:,ResourceVersion:,FieldPath:,}}